index.md

```diff
@@ -2,29 +2,34 @@
 layout: default
 ---
 
-## Content thats yours.
+# Olmec
 
-#### Buy. Sell. Resell. Rent. Lend. Collect.
+Olmec is a proof-of-concept in end-to-end token-gating.
 
-[Link to another page](./another-page.html).
+## End-to-end token-gating
 
-### Small image
+[End-to-end](https://en.wikipedia.org/wiki/End-to-end_principle) token-gated content is the next big step in web3.
 
-![Octocat](https://github.githubassets.com/images/icons/emoji/octocat.png)
+Most content in web2 is gated, from Netflix and YouTube to Amazon's ebooks and Instagram. As a creator, *you* are uploading *your* content to the server owned by a company. As a consumer, *you* are only buying temporary access to view the content or are paying by spending your time to watch ads.
 
-### Large image
+That will all change in web3. As a creator, *you* can own your own content and distribute them directly to your consumers. As a consumer, *you* can buy directly from consumers. You can also resell into a market, lend to friends, or collect them as memorabilia.
 
-![Branching](https://guides.github.com/activities/hello-world/branching.png)
+Remember when you could lend out a book to friends? You can't do that with ebooks on Amazon. Or resell a vinyl records to a store? You can't do that with a Spotify subscription. But that's all coming back but digitally.
 
-### Definition lists can be used with HTML syntax.
+NFTs are the way we get from here to there. But now, NFTs display metadata for all to see. With private NFTs, an NFT can unlock content if you own it. That's the beauty of end-to-end token-gating.
```

Comment:
Not sure how to say it better atm but I feel like the case for improving upon NFTs could be stronger.  Now it sounds just like we're motivating the Secret network, generally.  I'll think on it